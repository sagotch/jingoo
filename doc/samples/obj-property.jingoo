<dl>
  <dt>person.name</dt><dd>{{ person.name }}</dd>
  <dt>person.age</dt><dd>{{ person.age }}</dd>
  <dt>person["name"]</dt><dd>{{ person["name"] }}</dd>
  <dt>person["age"]</dt><dd>{{ person["age"] }}</dd>
</dl>
