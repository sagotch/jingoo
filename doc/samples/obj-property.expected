<dl>
  <dt>person.name</dt><dd>alice</dd>
  <dt>person.age</dt><dd>42</dd>
  <dt>person["name"]</dt><dd>alice</dd>
  <dt>person["age"]</dt><dd>42</dd>
</dl>
